lightning-tag/user-consent.md

```diff
@@ -0,0 +1,126 @@
+# User Consent Guide
+
+## Passing Consent Signals Into Lotame
+
+For clients that utilize Lotame's Consent Management Product, consent signals can be passed directly into Lightning Tag. Use this feature to inform Lotame of each user's consent preference and how they should be handled within the DMP.
+
+```javascript
+// Validate Lotame Lightning Tag script is ready
+if (window.lotame_<lotameClientId>.onTagReady() ) {
+
+  // Callback that runs when setConsent() completes that
+  //  allows webpage processing if needed at this event
+  function consentCb(returnData) {};
+
+  // Set these as true or false as per the signals from your User Consent process
+  var customerConsents = {
+    analytics: true,
+    crossdevice: true,
+    datasharing: true,
+    targeting: true
+  };
+  window.lotame_<lotameClientId>.setConsent(consentCb, <lotameClientId>, customerConsents);
+}
+```
+
+The full details of the `returnData` object passed to the callback can be found in the [Lightning Tag Field Reference Guide](lightning-tag/detailed-reference.md).
```

Comment:
To confirm this is referencing how the client can getConsent? I just want to make sure we have clear examples of how clients can use both setConsent & getConsent

Reply:
Added `getConsent`